person_capture/gui_app.py

```
@@ -10038,15 +10045,25 @@ def _side_gain(src_dev: np.ndarray, dst_dev: np.ndarray, side: int) -> Optional[
             except Exception:
                 lowfreq = 0.0
 
+            try:
+                default_black_chroma_neutralize = float(
+                    getattr(
+                        self.cfg,
+                        "hdr_wic_yuv444_color_match_black_chroma_neutralize",
+                        _WIC_YUV444_BLACK_CHROMA_NEUTRALIZE_DEFAULT,
+                    )
+                )
+            except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)